test/e2e/util.go

```diff
@@ -482,35 +539,49 @@ func RunRC(c *client.Client, name string, ns, image string, replicas int) error
 		return fmt.Errorf("Error creating replication controller: %v", err)
 	}
 
+	// Create a routine to query for the list of pods
+	stop := make(chan struct{})
+	go func(stop <-chan struct{}, n string, ns string, l labels.Selector) {
+		for {
+			select {
+			case <-stop:
+				return
+			default:
+				p, err := c.Pods(ns).List(l, fields.Everything())
+				if err != nil {
+					Logf("Warning: Failed to get pod list: %v", err)
+				} else {
+					podLists.Push(p.Items)
+				}
+				time.Sleep(1 * time.Second)
```

Comment:
@fgrzadkowski I understood that if I used an informer I would get events for all pod status changes.  This would include not just pod creation and start notifications, but also all the pod status updates.  If that is the case then using an informer would add a lot of complexity in the test to process all these events, many of which we won't care about.  What we want is a rolled up list of pod states.  Am I wrong?